backend/deploy.sh

```
@@ -0,0 +1,68 @@
+cd "$(dirname "$0")"
+
+# Read environment variables from .env.local (Robust sourcing strategy)
+if [ ! -f .env.local ]; then
+    echo "Error: .env.local not found. Please create it with GEMINI_API_KEY and FIREBASE_SERVICE_ACCOUNT_KEY."
+    exit 1
+fi
+
+# Load variables by sourcing the file
+if [ -f .env.local ]; then
+    set -a
+    source .env.local
```

Shellcheck (0.11.0)

[info] Not following: .env.local was not specified as input (see shellcheck -x).

(SC1091)